src/concrete/ml/sklearn/base.py

```diff
@@ -96,7 +102,10 @@
 # Enable rounding feature for all tree-based models by default
 # Note: This setting is fixed and cannot be altered by users
 # However, for internal testing purposes, we retain the capability to disable this feature
-os.environ["TREES_USE_ROUNDING"] = "1"
+os.environ["TREES_USE_ROUNDING"] = os.environ.get("TREES_USE_ROUNDING", "1")
+
+# By default, the decision of the tree ensembles is made in clear
+TREES_USE_FHE_SUM = False
```

Comment:
I think this is not needed anymore right ?

Reply:
Oh yes, forgot to remove it.
Thanks